### PR TITLE
Update message deletion info

### PR DESCRIPTION
**Pull request by jhaaaa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,21 +48,23 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
...
 
-This limitation will be mitigated very soon by the example application's UI, and resolved soon via improvements to the `xmtp-js` library that will allow messages to be created even if the intended recipient has not yet generated its keys.
+- The application hasn't undergone a formal security audit.
+- The application sends and receives messages using the XMTP `dev` network environment.
+     - **DO NOT** send sensitive information on the `dev` network.
+     - XMTP does not regularly delete messages and keys from the `dev` network. However, because XMTP is still in active development, we might need to delete messages and keys occasionally to maintain network stability. XMTP communicates about deletions in the XMTP Discord community ([request access](https://xmtp.typeform.com/to/yojTJarb?utm_source=docs_home)), providing as much advance notice as possible.
+- You can't yet send a message to a wallet address that hasn't used XMTP. The client displays an error when it looks up an address that doesn't have an identity broadcast on the XMTP network.
+   - This limitation will soon be communicated more clearly in the application UI.
```
Comment (jhaaaa) [inline README.md:69]: Is this already in place?

File: README.md [hunk 2]
```diff
@@ -6,15 +6,13 @@
...
 
-**This example chat application demonstrates the core concepts and capabilities of the XMTP Client SDK.** It is built with React, [Next.js](https://nextjs.org/), and the [`xmtp-js`](https://github.com/xmtp/xmtp-js) client library. The application is capable of sending and receiving messages via the [XMTP Labs](https://xmtp.com) development network, with no performance guarantees and [notable limitations](#limitations) to its functionality.
+**Example chat application demonstrating the core concepts and capabilities of the XMTP client SDK**
 
-It is maintained by [XMTP Labs](https://xmtp.com) and distributed under [MIT License](./LICENSE) for learning about and developing applications that utilize the XMTP decentralized communication protocol.
+This application is built with React, [Next.js](https://nextjs.org/), and the [`xmtp-js` client SDK](https://github.com/xmtp/xmtp-js).
 
-**All wallets and messages are forcibly deleted from the development network on Mondays.**
+Use the application to send and receive messages using the XMTP `dev` network environment, with some [important considerations](#considerations). You are also free to customize and deploy the application.
```
Comment (jhaaaa) [inline README.md:13]: Added the "customize and deploy" language from the documentation. Appropriate?

File: README.md [hunk 3]
```diff
@@ -50,21 +48,20 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
...
 2. A signature that is used on application start-up to initialize the XMTP client with that identity
 
 ### Chat Conversations
 
-The application utilizes the `xmtp-js` [Conversations](https://github.com/xmtp/xmtp-js#conversations) abstraction to list the available conversations for a connected wallet and to listen for or create new conversations. For each convesation, it gets existing messages and listens for or creates new messages. Conversations and messages are kept in a lightweight store and made available through **XmtpContext** alongside the client and its methods.
-
-### Limitations
-
-The application's functionality is limited by current work-in-progress on the `xmtp-js` client.
+The application uses the `xmtp-js` [Conversations](https://github.com/xmtp/xmtp-js#conversations) abstraction to list the available conversations for a connected wallet and to listen for or create new conversations. For each conversation, the application gets existing messages and listens for or creates new messages. Conversations and messages are kept in a lightweight store and made available through **XmtpContext** alongside the client and its methods.
```
Comment (saulmc) [inline README.md:58]:
```suggestion
The application uses the `xmtp-js` [Conversations](https://github.com/xmtp/xmtp-js#conversations) abstraction to list the available conversations for a connected wallet and to listen for or create new conversations. For each conversation, the application gets existing messages and listens for or creates new messages. Conversations and messages are kept in a lightweight store and made available through **XmtpProvider**.
```

File: README.md [hunk 4]
```diff
@@ -50,21 +48,20 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 
 [`Web3Modal`](https://github.com/Web3Modal/web3modal) is used to inject a Metamask, Coinbase Wallet, or WalletConnect provider through [`ethers`](https://docs.ethers.io/v5/). Methods for connecting and disconnecting are included in **WalletContext** alongside the provider, signer, wallet address, and ENS utilities.
```
Comment (saulmc) [inline README.md:49]:
```suggestion
[`Web3Modal`](https://github.com/Web3Modal/web3modal) is used to inject a Metamask, Coinbase Wallet, or WalletConnect provider through [`ethers`](https://docs.ethers.io/v5/). Methods for connecting and disconnecting are included in **WalletProvider** alongside the provider, signer, wallet address, and ENS utilities.
```

**Review by saulmc — APPROVED** [review]

**Pull request merged**